tests/integration/terraform/terraform.py

```diff
@@ -0,0 +1,110 @@
+"""Integration tests for Terraform."""
+
+import os
+import shutil
+import logging
+import subprocess
+from send2trash import send2trash
+from runway.util import change_dir
+
+WORKING_DIR = os.getcwd()
+TERRAFORM_TF_DIR = os.path.join(WORKING_DIR, 'test_terraform.tf')
+TERRAFORM_TESTS_DIR = os.path.join(WORKING_DIR, 'tests')
+TERRAFORM_STATE_DIR = os.path.join(WORKING_DIR, 'tf_state.cfn')
+
+LOGGER = logging.getLogger(__name__)
+STACKER_COMMANDS = ['build', 'destroy']
+
+
+def clean():
+    """Clean Terraform directories and any main.tf created."""
+    files_to_delete = ['main.tf', 'terraform.tfstate']
+    dirs_to_delete = ['.terraform', 'terraform.tfstate.d']
+
+    LOGGER.info('Running "runway destroy" ...')
+    with change_dir(WORKING_DIR):
+        subprocess.check_call('runway destroy', shell=True)
+
+    for file_to_delete in files_to_delete:
+        if os.path.isfile(os.path.join(TERRAFORM_TF_DIR, file_to_delete)):
+            LOGGER.debug('Trashing %s', file_to_delete)
+            send2trash(os.path.join(TERRAFORM_TF_DIR, file_to_delete))
+    for dir_to_delete in dirs_to_delete:
+        if os.path.isdir(os.path.join(TERRAFORM_TF_DIR, dir_to_delete)):
+            LOGGER.debug('Trashing %s', dir_to_delete)
+            send2trash(os.path.join(TERRAFORM_TF_DIR, dir_to_delete))
+
+
+def run_test(test_name):
+    """Rename test_name to main.tf and run runway."""
+    if not test_name.endswith('.test'):
+        test_name += '.test'
+
+    if os.path.isfile(os.path.join(TERRAFORM_TESTS_DIR, test_name)):
+        LOGGER.debug('Copying Terraform test: "%s" to main.tf', test_name)
+        shutil.copy(os.path.join(TERRAFORM_TESTS_DIR, test_name),
+                    os.path.join(TERRAFORM_TF_DIR, 'main.tf'))
+
+        LOGGER.info('Running test "%s"', test_name)
+        with change_dir(WORKING_DIR):
+            subprocess.check_call('runway deploy', shell=True)
```

Comment:
Does this work instead:
```
subprocess.check_call(['runway', 'deploy'])
```
?
(to get rid of the `shell=True`)

Reply:
Yep - this works.